Packs/PICUS/Integrations/PICUS/PICUS.py

```diff
@@ -0,0 +1,901 @@
+            "result_time": "2018-08-21T13:00:33.590599407Z",
+            "variant": "HTTP"
+          }
+        ],
+        "is_disabled": true,
+        "name": "Email-Peer-2 - Email-Peer-1",
+        "trusted": "Windows10-Peer",
+        "type": "string",
+        "untrusted": "Network-Peer",
+        "users": [
+          {
+            "role": "admin",
+            "username": "oliver"
+          }
+        ]
+      }
+    ]
+  },
+  "success": true
+}
+'''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    ''' EXECUTION '''
+    #LOG('command is %s' % (demisto.command(), ))
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+        LOG('Command being called is {command}'.format(command=demisto.command()))
+        if demisto.command() == 'Picus-GetAccessToken':
+            getAccessToken()
+        elif demisto.command() == 'Picus-Vector-Compare':  # Makes a comparison of the given vector's results
+            token = getAccessToken()
+            demisto.results(vectorCompare(token))
+        elif demisto.command() == 'Picus-Attack-Result-List':  # Returns the list of the attack results\nhave optional parameters for pagination and filtration
+            token = getAccessToken()
+            demisto.results(attackResultList(token))
+        elif demisto.command() == 'Picus-Specific-Threats-Results':  # Returns the list of the attack results of a single threat\nhave optional
+            token = getAccessToken()
+            demisto.results(specificThreatsResults(token))
+        elif demisto.command() == 'Picus-Peer-List':  # Returns the peer list with current statuses
+            token = getAccessToken()
+            demisto.results(peerList(token))
+        elif demisto.command() == 'Picus-EMail-Peer-List':  # Returns the E-Mail peer list with current statuses
+            token = getAccessToken()
+            demisto.results(eMailPeerList(token))
+        elif demisto.command() == 'Picus-Attack-All-Vectors':  # Schedules given attack on all possible vectors
+            token = getAccessToken()
+            demisto.results(attackAllVectors(token))
+        elif demisto.command() == 'Picus-Attack-Single':  # Schedules a single attack on requested vector
+            token = getAccessToken()
+            demisto.results(attackSingle(token))
+        elif demisto.command() == 'Picus-Trigger-Update':  # Triggers the update mechanism manually, returns if the update-command is taken successfully
+            token = getAccessToken()
+            demisto.results(triggerUpdate(token))
+        elif demisto.command() == 'Picus-Version':  # Returns the current version and the update time config
+            token = getAccessToken()
+            demisto.results(version(token))
+        elif demisto.command() == 'Picus-Threat-List':  # Returns the list of the threats\nhave optional parameters for pagination and filtration
```

Comment:
Notice that the YML doesn't state that there is a command like that.
In case you would like to add it to the YML, use the name convention below.

```suggestion
        elif demisto.command() == 'picus-threat-list':  # Returns the list of the threats\nhave optional parameters for pagination and filtration
```

Reply:
Resolved.